### ISSUE TITLE
Can you share the details of how you collected the demographics data?

### ISSUE THREAD

**Issue by abaheti95** [wri/demographic-identifier#4]
I was looking at the `data/processed/1-demographics/` folder which contains some of the user annotated data. Can you describe how did you collect and label this data? Also what kind of filtering did you apply on this data? Is there a paper or report where I can read the details of the training data?

Thanks in advance!

**Comment by JohnMBrandt** [comment]
Hi -

Sorry for the late response!
This work was published in the journal of computational social science, here:
https://link.springer.com/article/10.1007/s42001-019-00061-9

The paper talks about the collection and labeling of data, as well as the filtering of the data to remove noise
